### PR TITLE
Update admin docs

### PR DESCRIPTION
**Pull request by Hexcles**
* Add an emergency playbook in the wake of #21424
* Update contacts by the way

### CODE REVIEW

File: docs/admin/index.md [hunk 1]
```diff
@@ -72,3 +74,21 @@ explicitly-managed secret.
 
 [web-platform-tests]: https://github.com/e3c/web-platform-tests
 [wpt.fyi]: https://github.com/web-platform-tests/wpt.fyi
+
+## Emergency playbook
+
+### Lock down write access to the repo
+
+**Recommended but not yet verified approach:** Create a[new branch protection
```
Comment (Hexcles) [inline docs/admin/index.md]:
```suggestion
**Recommended but not yet verified approach:** Create a [new branch protection
```

File: docs/admin/index.md [hunk 2]
```diff
@@ -72,3 +74,21 @@ explicitly-managed secret.
...
+matching branches". This should prevent everyone except those with the
+"Maintain" role (currently only the GitHub admins listed above) from pushing
+to *any* branch. To lift the limit, delete this branch protection rule.
+
+**Alternative apporach proven to work in [#21424]
+(https://github.com/web-platform-tests/wpt/issues/21424):** Go to [manage
+access](https://github.com/web-platform-tests/wpt/settings/access), and
+change the permission of "reviewers" to "Read". To lift the limit, change it
+back to "Write". This has the known downside of *resubscring all reviewers
+to repo notifications*.
```
Comment (Hexcles) [inline docs/admin/index.md]: **Alternative apporach proven to work in
[#21424](https://github.com/web-platform-tests/wpt/issues/21424):** Go to
[manage access](https://github.com/web-platform-tests/wpt/settings/access),
and change the permission of "reviewers" to "Read". To lift the limit, change
it back to "Write". This has the known downside of *resubscring all reviewers
to repo notifications*.
```

File: docs/admin/index.md [hunk 3]
```diff
@@ -86,9 +86,9 @@ matching branches". This should prevent everyone except those with the
...
-**Alternative apporach proven to work in [#21424]
-(https://github.com/web-platform-tests/wpt/issues/21424):** Go to [manage
-access](https://github.com/web-platform-tests/wpt/settings/access), and
-change the permission of "reviewers" to "Read". To lift the limit, change it
-back to "Write". This has the known downside of *resubscring all reviewers
+**Alternative apporach proven to work in
+[#21424](https://github.com/web-platform-tests/wpt/issues/21424):** Go to
+[manage access](https://github.com/web-platform-tests/wpt/settings/access),
+and change the permission of "reviewers" to "Read". To lift the limit, change
+it back to "Write". This has the known downside of *resubscring all reviewers
```
Comment (sideshowbarker) [inline docs/admin/index.md]:
```suggestion
it back to "Write". This has the known downside of *resubscribing all reviewers
```

File: docs/admin/index.md [hunk 4]
```diff
@@ -86,9 +86,9 @@ matching branches". This should prevent everyone except those with the
 "Maintain" role (currently only the GitHub admins listed above) from pushing
 to *any* branch. To lift the limit, delete this branch protection rule.
 
-**Alternative apporach proven to work in [#21424]
-(https://github.com/web-platform-tests/wpt/issues/21424):** Go to [manage
-access](https://github.com/web-platform-tests/wpt/settings/access), and
-change the permission of "reviewers" to "Read". To lift the limit, change it
-back to "Write". This has the known downside of *resubscring all reviewers
+**Alternative apporach proven to work in
```
Comment (sideshowbarker) [inline docs/admin/index.md]:
```suggestion
**Alternative approach proven to work in
```

**Review by sideshowbarker — APPROVED** [review]
Typos

**Comment by Hexcles** [comment]
Oops... And thanks!

**Pull request merged**